client/src/views/logs.tsx

```diff
@@ -77,9 +81,38 @@ export default class Logs extends Base<Props, State> {
         });
     }
 
+    startDownloadLog = () => {
```

Comment:
I know this style of function declaration is popular and I totally get the reasoning. However, we've used the more traditional `startDownloadLog() {` format everywhere else in the code base. For consistency sake, I think we should stick with that throughout (at least for the time being).

Reply:
FIxed